Source/ui_macosx/ApplicationDelegate.mm

```diff
@@ -100,6 +103,28 @@ -(IBAction)showPreferences: (id)sender
 	[self setupSoundHandler];
 }
 
+-(IBAction)screenCaptureMenuSelected: (id)sender
+{
+	static_cast<CGSH_OpenGL*>(g_virtualMachine->m_ee->m_gs)->OnFlipImplComplete.connect_extended(
```

Comment:
so are you suggesting to create a virtual function in `GSHandler`? (as I'm not sure how generic the current method can get with `glReadPixels()` being OpenGL specific)